### DISCUSSION TITLE
Workflow | Update field | UI10

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16738] [How to] [answered]
Is there any trick to update reference field via workflow/Update fields with raw text, but so it automatically match existing record?
I mean workflow raw text has exact text string as record entity field in referenced module

best

**Comment by rskrzypczak** [comment, marked as answer]
The only trick here is that you can pass the ID record (not Number) instead of its name.

**Reply by s2b-git** [reply]
Doesn't work either. Should I use some certain format of ID?

**Reply by s2b-git** [reply]
apologize. It works. Messed with conditions
